desktopAssistant.py

```diff
@@ -32,7 +32,7 @@ def myCommand():
 
     #loop back to continue to listen for commands if unrecognizable speech is received
     except sr.UnknownValueError:
-        assistant(myCommand())
+        myCommand()

```

Comment:
I think this handling of exception isn't really required. What I mean is a simple pass statement would work just fine, if an exception occurs the function will end and the loop statement on line 83 would pick thigs up.